src/index.html

```diff
@@ -1,13 +1,468 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>Crazybaby</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="icons/favicon2.svg"
+    >
+    <link rel="stylesheet" href="styles/styles.scss">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+  <body class="page__body">
+    <header class="header" id="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#" class="top-bar-logo">
+              <img
+                src="./icons/logo.svg"
+                alt="Crazylab logo"
+                class="header__logo"
```

Comment:
`header__logo` is an element of the parent `header`
`top-bar-logo` is an element or block? and `top-bar-logo` can not have element `header__logo` without parent `header`